### DISCUSSION TITLE
ModuleNotFoundError: No module named 'django_celery_results'

### DISCUSSION THREAD

**Discussion by vismoh2010** [celery/django-celery-results#393] [Q&A] [answered]
This happens when I try to make migrations. I tried doing `pip uninstall django-celery-results` and `pip install django-celery-results` again. I have `"django_celery_results",` in `INSTALLED_APPS` in my settings.py file. What do I do?

**Comment by rimlol** [comment]
Having the same problem. Anybody have an idea how to fix?

**Reply by collindutter** [reply, marked as answer]
In my case it was a missing comma in `INSTALLED_APPS` as suggested [here](https://github.com/celery/django-celery-results/issues/318).
